Gruntfile.js

```diff
@@ -13,6 +13,14 @@ module.exports = function(grunt) {
 					'composer.json',
 					'CHANGELOG.md',
 					'LICENSE.md',
+
+					/*
+					 * Exclude files not necessary in the distribution.
+					 *
+					 * @link https://github.com/liquidweb/airstory-wp/issues/69
+					 */
+					'!includes/lib/wp-async-task/*.dist',
```

Comment:
Blocking it in the build step and adding it to the .svnignore file would work. Nothing says the two have to match exactly.

I have seen .svnignore files dropped into a git repo when dealing with WordPress VIP deployments, so I think it could be valid either way. As long as we have the svn repo set up in a way that blocks the files from there if someone fails to run the build or accidentally deletes this line, I'm good with it.